index.html

```diff
@@ -1,122 +1,120 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <title>Fork Me! FCC: Test Suite Template</title>
-    <link rel="stylesheet" href="css/style.css">
-    <script src="https://kit.fontawesome.com/63ff2cfeab.js" crossorigin="anonymous"></script> 
-  
-  </head>
 
-  <body>
-    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-    
-     <header id="header">
-       <nav id="navbar">
-         <a href="#">About</a>
-         <a href="#">Work</a>
-         <a href="#">Contact</a>
-       </nav>
-     </header>
-     <div id="welcome-section">
-       <h1>Hey I am Mimic</h1>
-       <h3>a web developer</h3>
-     </div>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+  <script src="https://kit.fontawesome.com/63ff2cfeab.js" crossorigin="anonymous"></script>
 
-    <div id="project-section">
-      <h1>These are some of my projects</h1>
-      <div id="projects">
-      <div class="project-tile" id="project">
-          <a href="#">
-            <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" >
+</head>
+
+<body>
+  <header id="header">
+    <nav id="navbar">
+      <a href="#">About</a>
+      <a href="#">Work</a>
+      <a href="#">Contact</a>
```

Comment:
You should use <li> tags for navbar links like so
```suggestion
      <li> <a href="#"> About </a> </li>
```

Reply:
Try to be more descriptive with your classes and id's. Try not to repeat div and don't abuse of <h1><h2><h3> and so on tags.

Reply:
- [ ] I would recommend phrasing the comments politely, as suggestions instead of orders.
For example:
This is a polite suggestion: "I think it would be better to use semantic tags"
This reads as an order: "Use semantic tags"